### ISSUE TITLE
Untie bonus maps and commentary buttons from game directory

### ISSUE THREAD

**Issue by Luke18033** [Joshua-Ashton/HL2-GamepadUI#1]
The commentary and bonus maps buttons in the new game dialog are [currently](https://github.com/Joshua-Ashton/HL2-GamepadUI/blob/1799abad208daa3992e93f65baaf2d7a83cff275/src/game/gamepadui/gamepadui_newgame.cpp#L31) tied to hardcoded game directory strings, meaning they can never show up in mods. With this now being open source it's less of a concern because modders can edit the code to make the buttons always appear, but it'd still be nice to have the ability to enable them without having to compile a custom DLL.

It'd probably make the most sense to have it controlled by 2 booleans in `schemechapterbutton.res`, since that file is already loaded by the new game dialog.

**Comment by DasMaddi** [comment]
I can see the use-cases for this, however, in order to best preserve backwards compatibility with those titles, I would want to have the scheme checks only override the directory ones, instead of the scheme entry being the only on-off switch, if that makes sense. @Joshua-Ashton any thoughts?

**Comment by Luke18033** [comment]
Yeah that's fine, the important thing is just giving mods a way to enable the buttons
